packages/flutter_tools/lib/src/resident_devtools_handler.dart

```diff
@@ -49,7 +47,7 @@ class FlutterResidentDevtoolsHandler implements ResidentDevtoolsHandler {
 
   static const Duration launchInBrowserTimeout = Duration(seconds: 15);
 
-  final DevtoolsLauncher _devToolsLauncher;
+  final DevtoolsLauncher? _devToolsLauncher;
```

Comment:
would it be valid for this to be null?

Reply:
we have tests that assert that the flutter daemon works when this is null

Reply:
:+1: